.github/workflows/CHERRY_PICK.md

```diff
@@ -0,0 +1,49 @@
+# Automated Cherry-Pick Workflow
+
+## Usage
+
+### 1. Using Labels (Recommended)
+Add labels to your PR before merging:
+- `cherry-pick/release-2.29.1-ccip`
+- `cherry-pick/release-cre`
+
+The cherry-pick will automatically trigger when the PR is merged to `develop`.
+
+### 2. Using Comments
+Comment on any merged PR:
+```
+pick release-cre, release-2.29.1-ccip
+```
+
+### 3. Manual Cherry-Pick
+```bash
+# From repository root
+./.github/scripts/cherry-pick-helper.sh <commit-sha> <target-branch>
```

Comment:
I pushed 3e898c4bda to make cherry-pick-helper.sh executable so we can run it like this.